errors/X509_V_ERR_UNSPECIFIED/data.yml

```diff
@@ -1,4 +1,5 @@
 tags: [ openssl, uncategorized ]
 openssl-docs: |
   Unspecified error; should not happen.
+unused-openssl: |
```

Comment:
The key name is inconsistent with the others. we use the library name as a prefix, not a suffix (see `openssl-docs`, `gnutls-docs` and similar). I thus suggest `openssl-unused`.
Secondly, the empty multiline string denoted by the pipe (`|`) seems a bit counterintuitive. I would prefer `openssl-unused: true`.